MediaBrowser.Api/UserLibrary/ItemsService.cs

```diff
@@ -213,7 +213,10 @@ private QueryResult<BaseItem> GetQueryResult(GetItems request, DtoOptions dtoOpt
                 request.IncludeItemTypes = "Playlist";
             }
 
-            bool isInEnabledFolder = user.Policy.EnabledFolders.Any(i => new Guid(i) == item.Id);
+            bool isInEnabledFolder = user.Policy.EnabledFolders.Any(i => new Guid(i) == item.Id) ||
```

Comment:
Boolean operators should be at the beginning of the next line

Reply:
Done!